context/app/routes_main.py

```diff
@@ -65,17 +66,19 @@ def details(type, uuid):
                                  'issue_url': error.issue_url,
                                  'traceback': error.__str__()[0:1500]})
 
-    template = f'pages/details_react.html'
-    props = {
+    template = f'pages/base_react.html'
+    core_props = {'endpoints': {'esEndpoint': current_app.config['ELASTICSEARCH_ENDPOINT']}}
+    core_props.update({
         'flashed_messages': flashed_messages,
         'entity': entity,
         'provenance': provenance,
         'vitessce_conf': client.get_vitessce_conf(),
-    }
+    })
     return render_template(
-        template, type=type, uuid=uuid,
-        title_text='TODO: title_text',
-        flask_data=props
+        template,
+        type=type,
+        uuid=uuid,
+        flask_data=core_props
```

Comment:
`core_props` could be moved out to the top level, and we don't need to mutate it.
```suggestion
        flask_data={
            'flashed_messages': flashed_messages,
            'entity': entity,
            'provenance': provenance,
            'vitessce_conf': client.get_vitessce_conf(),
            **core_props
        }
```